tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -1072,7 +1072,7 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
       |  override def close : Unit = {}
       |}
       |class C extends B{
-      |  clos@@
+      |  close@@
```

Comment:
For now, we don't provide override completions in scala 3 on incomplete symbol names without `override` or `def` keyword, I will on that next so it works like in scala 2

Reply:
Let's maybe have it as a separate test? One with `clos` and the other with `close?`

Reply:
I've added both test and a fix for that